README.md

```diff
@@ -1 +1,85 @@
-# react-capstone
\ No newline at end of file
+
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# COVID-19 Tracker
+
+Covid-19 Tracker is a mobile web application that provide the user the new cases of Covid-19 for each country and also the regions within the country 
+by getting all the data and the statistics from [Narrativa](https://covid19tracking.narrativa.com/index_en.html).
+
+The web app consists of Home, Countries List, and the Regions List page.The original design idea belongs to Nelson Sakwa. [Link to the original design.](https://www.behance.net/gallery/31579789/Ballhead-App-%28Free-PSDs%29)
```

Comment:
- [ ] Great job :+1:  however the appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), should be given in the "Acknowledgements" section of your README.md file as required by the [Creative Commons License](https://creativecommons.org/licenses/). [](https://github.com/jaflih/metrics/pull/24/files#live-version), Kindly add this, in the meantime I'm going to approved this project since it's the only thing you miss :+1:

Reply:
Thanks for the highlight, I will work on it